Palace.xcodeproj/project.pbxproj

```diff
@@ -3836,7 +3836,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.0.16;
+				MARKETING_VERSION = 1.0.17;
```

Comment:
we haven't published 1.0.16 yet, could you revert to 1.0.16 and increase build number only?